src/cypress-runner.ts

```diff
@@ -208,6 +209,13 @@ async function canAccessFolder(file: string) {
   await fsAccess(file, fs.constants.R_OK | fs.constants.W_OK);
 }
 
+function zipArtifacts(runCfg: RunConfig) {
+  Object.keys(runCfg.artifacts.remain).forEach((key) => {
+    const value = runCfg.artifacts.remain[key];
```

Comment:
See the other comment below:
```suggestion
  Object.keys(runCfg.artifacts.retain).forEach((key) => {
    const value = runCfg.artifacts.retain[key];
```